tests/chainerx_tests/unit_tests/routines_tests/test_misc.py

```diff
@@ -199,7 +199,13 @@ def test_sign(xp, device, input, dtypes):
 ))
 class TestMaximum(math_utils.BinaryMathTestBase, op_utils.NumpyOpTest):
 
-    dodge_nondifferentiable = True
+    def generate_inputs(self):
+        for _ in range(10):
+            a, b = super().generate_inputs()
+            if ((a.dtype.kind in 'biu' and b.dtype.kind in 'biu') or
+                    (numpy.abs(a - b) > 0.01).all()):
+                return a, b
+        assert False
```

Comment:
Just a nitpick but how avoid making the error more expressive?
I am not sure wether we allow to add error messages to the assert.
If not I believe its self-descriptive enough.